robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowAccessibilityEvent.java

```diff
@@ -21,6 +24,9 @@
   private static final Map<StrictEqualityEventWrapper, StackTraceElement[]> obtainedInstances =
       new HashMap<>();
 
+  private static final SparseArray<StrictEqualityEventWrapper> orderedInstances = new SparseArray<>();
```

Comment:
This is the SparseArray nonsense where it uses == instead of equals() for key lookup.  WAI after the removal of decrements to sAllocationCount.